scripts/render-notebooks.R

```diff
@@ -107,13 +107,17 @@ new_lines <- append(lines, header_line, header_range[1])
 # Write to an tmp file
 readr::write_lines(new_lines, tmp_file)
 
+# Declare path to google analytics bit
+google_analytics_file <- normalizePath(file.path("components", "google-analytics.txt"))
```

Comment:
It may not look like it, but this is an .html file, and should get that extension.

```suggestion
google_analytics_file <- normalizePath(file.path("components", "google-analytics.html"))
```